tests/run-make/raw-dylib-elf/rmake.rs

```diff
@@ -0,0 +1,24 @@
+//@ only-elf
```

Comment:
The `x86_64-fortanix-unknown-sgx` target doesn't support dynamic linking. Can you please add `//@ ignore-sgx` to ignore this platform?